src/js/index.js

```diff
@@ -119,35 +119,40 @@ async function init() {
     pc.profile = data.id;
     pc.config.baseUrl = pc.baseUrl;
     if (GOOGLE_ANALYTICS_ID !== "undefined" && GOOGLE_ANALYTICS_ID !== "") {
-        const head = document.getElementsByTagName('head')[0];
+        pc.config.analytics = initGA(GOOGLE_ANALYTICS_ID, pc.profile);
+    } else {
+        // dev or staging
+        pc.config.analytics = initGA('UA-93649482-33', pc.profile);
+    }
+    pc.config.profile = data.id;
```

Comment:
that could've been simpler but making it simpler now would take time and since it is not new logic I don't think it is in the scope of current task @goyal1092

Reply:
Cool, Sounds good 👍